docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -113,6 +123,19 @@ The following configuration properties are available:
      - The field of the access token used for the Trino user principal. Defaults to ``sub``. Other commonly used fields include ``sAMAccountName``, ``name``, ``upn``, and ``email``.
    * - ``http-server.authentication.oauth2.groups-field``
      - The field of the access token used for Trino groups. The corresponding claim value must be an array.
+   * - ``http-server.authentication.oauth2.oidc.discovery``
+     - Enable Open ID provider
+       `metadata <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata>`_
+       reading. Default is ``true``.
```

Comment:
Since discovery is enabled by default, can/should we simplify the example in line 43? Having a minimum example might make it easier for users to configure OAuth when they might not be very experienced with it before they go looking for the values for all options only to learn later that they're not required. If we need an example for providers that don't support discovery, there should be another one, with some comment about what's OIDC discovery.

Reply:
Simplified the example. I believe most of OAuth2 complaint providers are OIDC complaint but a dedicated section for them seems to make sense. What do you think? @mosabua @kokosing